project-progression-policy.md

```diff
@@ -151,7 +160,7 @@ Projects in this stage are not in active development. Their maintainers may infr
 
 **Acceptance Criteria**
 
-Projects may be granted Emeritus status via a vote of two-thirds of the representatives of the  TAC.
+Projects may be granted Emeritus status via a vote of two-thirds of the representatives of the TAC.
 
 
 ## IV. Annual Review Process
```

Comment:
I think before we accept this change we should actually define the annual review process. This needs to be a subject w discuss at a TAC meeting. What are we looking for the Review Process to achieve?